src/lib/getTxLink.ts

```diff
@@ -32,11 +52,32 @@ export const getTxBaseUrl = ({ name, defaultExplorerBaseUrl, isOrder }: GetTxBas
   }
 }
 
-export const getTxLink = ({ name, defaultExplorerBaseUrl, txId, tradeId }: GetTxLink): string => {
+export const getTxLink = ({
+  name,
+  defaultExplorerBaseUrl,
+  txId,
+  tradeId,
+  isSafeTxHash,
+  accountId,
+}: GetTxLink): string => {
   const id = txId ?? tradeId
   const isOrder = !!tradeId
   const baseUrl = getTxBaseUrl({ name, defaultExplorerBaseUrl, isOrder })
 
+  // TODO(gomes): make this async, pass full serializedTxid (including address and AccountId, hence ChainId), and construct a correct link in the form of
+  // https://app.safe.global/transactions/tx?id=multisig_<safeAddy>_<safeTxHash>&safe=avax:<safeAddy>
+  // where avax is the prefix of the chain
+  // Alternatively, only pass AccountId and ChainId to avoid deserialization madness
```

Comment:
i would say easier to pass `accountId` with and not do the serialization back and forth businees everywhere and delete this comment. good as is imo

Reply:
Keeping the TODO to be tackled in  https://github.com/shapeshift/web/pull/7525 if you don't mind @kaladinlight!

Reply:
Sounds good!